frontend/src/features/permits/pages/Application/ApplicationForm.tsx

```diff
@@ -190,8 +190,7 @@ export const ApplicationForm = ({
   // When "Continue" button is clicked
   const onContinue = async (data: ApplicationFormData) => {
     const updatedViolations = await triggerPolicyValidation();
-    // prevent CV client continuing if there are policy engine validation errors
-    if (Object.keys(updatedViolations).length > 0 && !isStaffUser) {
+    if (Object.keys(updatedViolations).length > 0) {
```

Comment:
Isn't this the change you made recently? Is it correct to remove it here?

Reply:
that is correct, must have accidentally removed this when merging main - added back in